.github/workflows/cwag-workflow.yml

```diff
@@ -0,0 +1,220 @@
+---
+name: cwag-workflow
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+    paths:
+      - 'orc8r/**'
+      - 'lte/**'
+      - 'feg/**'
+      - 'cwf/**'
```

Comment:
We still want to trigger most of these jobs on the pull request event.
On pull request event we will not trigger the `xwfm-test` and push to artifactory.
But we should still trigger the rest.
To keep the dependency in `xwfm-deploy-latest`, you can try to skip the whole `xwfm-test` job. (Or all of the steps if it does not work)

Reply:
thank you!.. let me push one possible solution now without the paths and see how it looks..